### PR TITLE
feat(screenshare, avatar) - show screen share tile placeholder when sharing screen is started

### PR DESCRIPTION
**Pull request by dudumanbogdan**
- show screen share tile placeholder when currentLayout is not `TILE_VIEW`

### CODE REVIEW

File: react/features/filmstrip/components/web/Thumbnail.tsx [hunk 1]
```diff
@@ -198,6 +206,16 @@ export interface IProps {
...
 
+    /**
+     * Whether or not to display virtual screen share placeholder.
+     */
+    _shouldDisplayVirtualScreenSharePlaceholder: boolean;
+
+    /**
+     * Whether source name signaling is enabled.
+     */
+    _sourceNameSignalingEnabled: boolean;
```
Comment (saghul) [inline react/features/filmstrip/components/web/Thumbnail.tsx:217]: This is no longer necessary. Please rebase.

File: react/features/base/avatar/components/Avatar.js [hunk 2]
```diff
@@ -208,7 +220,7 @@ class Avatar<P: Props> extends PureComponent<P, State> {
             avatarProps.url = effectiveURL;
         }
 
-        const initials = getInitials(_initialsBase);
+        const initials = ignoreInitials ? null : getInitials(_initialsBase);
```
Comment (saghul) [inline react/features/base/avatar/components/Avatar.js:223]: Why do we want to ignore the initials? It would be nice to get the avatar color to match the participant it's related to I guess.

File: react/features/base/avatar/functions.ts [hunk 3]
```diff
@@ -77,5 +77,5 @@ export function getInitials(s?: string) {
  * @returns {void}
  */
 export function isCORSAvatarURL(url: string, corsURLs: Array<string> = []): boolean {
-    return corsURLs.some(pattern => url.startsWith(pattern));
+    return typeof url === 'string' && corsURLs.some(pattern => url.startsWith(pattern));
```
Comment (saghul) [inline react/features/base/avatar/functions.ts:80]: Why is this necessary?

**Review by saghul — COMMENTED** [review]
Some quick comments.

File: react/features/base/avatar/components/Avatar.js (hunk 2 above)
Comment (dudumanbogdan) [reply react/features/base/avatar/components/Avatar.js:223]: I got from UX feedback that we want only one color to be displayed for placeholder with the icon in the middle, no initials on it; I will raise this and if we want to match the colors of the participant, I would need to only skip this part `avatarProps.initials = initials;`

File: react/features/base/avatar/functions.ts (hunk 3 above)
Comment (saghul) [reply react/features/base/avatar/functions.ts:80]: The we also need to update the type anotation

**Comment by dudumanbogdan** [comment]
Closed the PR as the UX req is different now. New PR for this https://github.com/jitsi/jitsi-meet/pull/12584
